kpi/views/v2/data.py

```diff
@@ -556,9 +556,11 @@ def duplicate(self, request, pk, *args, **kwargs):
         deployment = self._get_deployment()
         # Coerce to int because back end only finds matches with same type
         submission_id = positive_int(pk)
+        original_submission = deployment.get_submission(submission_id, request.user)
```

Comment:
It seems, we only need the `original_submission['_uuid']`, thus one optimisation could be 

```python
original_submission = deployment.get_submission(
    submission_id, 
    request.user, 
    fields=['_uuid']
)
```

to make MongoDB return only `_uuid`

Reply:
Ok, sounds good